src/components/Button/Base.js

```diff
@@ -1,9 +1,20 @@
 import React from "react";
 import PropTypes from "prop-types";
+import { ThemeProvider } from "styled-components";
 
 import { StyledButton, StyledButtonLink } from "./Base.styles";
 import { BUTTON_VARIANTS, BUTTON_SIZES } from "./constants";
 import getRelByTarget from "../../utils/link";
+import themes from "../../theme/colorThemes";
```

Comment:
It is better to import from index.js in theme folder

Reply:
fixed